instaloader/__main__.py

```diff
@@ -66,6 +67,45 @@ def filterfunc(item) -> bool:
 
     return filterfunc
 
+def get_cookies_from_instagram(domain, browser, cookiefile="", cookieName=''):
+    supported_browsers = {
+        "chrome": browser_cookie3.chrome,
+        "firefox": browser_cookie3.firefox,
+        "edge": browser_cookie3.edge,
+        "brave": browser_cookie3.brave,
+        "opera": browser_cookie3.opera,
+        "safari": browser_cookie3.safari
+    }
+
+    if browser not in supported_browsers:
+        print("Loading cookies from the specified browser failed. Supported browsers are Chrome, Firefox, Edge, Brave, Opera and Safari")
```

Comment:
This line is very long, we have set a maximum of 120 characters.

Reply:
I have broken down this line.